docs/logs/README.md

```diff
@@ -0,0 +1,105 @@
+# OpenTelemetry .NET Logs
+
+## Best Practices
+
+The following tutorials have demonstrated the best practices for logging with
+OpenTelemetry .NET:
+
+* [Getting Started - Console Application](./getting-started-console/README.md)
+* [Getting Started - ASP.NET Core
+  Application](./getting-started-aspnetcore/README.md)
+* [Logging with Complex Objects](./complex-objects/README.md)
+
+## Structured Logging
+
+:heavy_check_mark: You should use structured logging.
+
+* Structured logging is more efficient than unstructured logging.
+  * Filtering and redaction can happen on invidual key-value pairs instead of
+    the entire log message.
+  * Storage and indexing are more efficient.
+* Structured logging makes it easier to manage and consume logs.
+
+:stop_sign: You should avoid string interpolation.
+
+> [!WARNING]
+> The following code has bad performance due to [string
+  interpolation](https://learn.microsoft.com/dotnet/csharp/tutorials/string-interpolation):
+
+```csharp
+var food = "tomato";
+var price = 2.99;
+
+logger.LogInformation($"Hello from {food} {price}.");
+```
+
+Refer to the [logging performance
+benchmark](../../test/Benchmarks/Logs/LogBenchmarks.cs) for more details.
+
+## Package Version
+
+:heavy_check_mark: You should always use the
+[`ILogger`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger)
+interface from the latest stable version of
+[Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging/)
+package, regardless of the .NET runtime version being used:
```

Comment:
Might be good to explicitly bless the usage of `ILogger<T>` here as well? Don't want people to think they shouldn't use it 🤷

You should always use the [`ILogger`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger) or [`ILogger<TCategoryName>`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger-1) interfaces...

Reply:
Next PR. I plan to cover how to retrieve the logger (DI, LoggerFactory) and lifecycle management.